src/api/database/bulk-api.rst

```diff
@@ -456,6 +456,132 @@ Sending multiple queries to a database
     The multiple queries are also supported in /db/_local_docs/queries and
     /db/_design_docs/queries (similar to /db/_all_docs/queries).
 
+.. _api/db/bulk_get:
+
+==================
+``/db/_bulk_get``
+==================
+
+.. http:post:: /{db}/_bulk_get
+    :synopsis: Fetches several documents at the given revision
```

Comment:
What do you mean by "at the given revision"?  If I'm not mistaken, this method (like most others) defaults to the current revision, and specifying the revision is optional?  If my understanding is accurate, I suggest simplifying the description to just:

> Fetches several documents simultaneously

Reply:
It's true, but if you just want to fetch several documents simultaneously, you should probably use `/db/_all_docs` that is simpler.

Reply:
Mmm, that's a fair point. :)

In that case, I suggest simply changing "revision" to "revisions".

Reply:
It's done